src/main/java/list.java

```diff
@@ -0,0 +1,65 @@
+public class list {
+    private static String[] l = new String[100];
+    public static int listSize = 0;
+    private static String[] isDone = new String[100];
+    private static String[] type = new String[100];
+
+    public static void initializing() {
+        for (int i = 0; i < 100; i++) {
+            isDone[i] = " ";
+            type[i] = " ";
+        }
+    }
+
+    public static void addToList(String input) {
+        if (input.contains("todo")) {
+            l[listSize] = input.substring(5);
+            type[listSize] = "T";
+        } else if (input.contains("deadline")) {
+            String by = input.substring(input.indexOf("/") + 4);
+            l[listSize] = input.substring(9, input.indexOf("/")) + " (by: " + by
```

Comment:
Avoid having complex nested logic like this. Either separate it out into different lines or abstract it out into different functions